core/CMakeLists.txt

```diff
@@ -767,27 +772,12 @@ if (NOT APPLE)
     if (NOT EXISTS ${CMAKE_OBJCOPY})
       message( "dynamorio_static requires objcopy")
     endif ()
-    get_target_property(dynamorio_dot_a dynamorio_static LOCATION)
-    string(REGEX REPLACE "\\.a$" ".o" dynamorio_dot_o "${dynamorio_dot_a}")
-    string(REPLACE " " ";" partial_link_flags "${CMAKE_C_FLAGS}")  # Get -m32/64
-    set (globalize_pic_thunks)
-    if (NOT X64)
-      # Localizing PIC thunks on ia32 creates references to discarded sections.
-      # XXX: Hardcoding this list of symbols is fragile.  It is known to work
-      # with gcc 4.4 and 4.6.  gcc 4.7 uses symbols starting with __x86.
-      set (globalize_pic_thunks ${CMAKE_OBJCOPY}
-        --globalize-symbol=__i686.get_pc_thunk.bx
-        --globalize-symbol=__i686.get_pc_thunk.cx
-        --globalize-symbol=__x86.get_pc_thunk.ax
-        --globalize-symbol=__x86.get_pc_thunk.bx
-        --globalize-symbol=__x86.get_pc_thunk.cx
-        --globalize-symbol=__x86.get_pc_thunk.dx
-        --globalize-symbol=__x86.get_pc_thunk.di
-        --globalize-symbol=__x86.get_pc_thunk.si
-        --globalize-symbol=__x86.get_pc_thunk.bp
-        "${dynamorio_dot_o}" &&
-        )
-    endif ()
+
+    set(locvar_name dynamoriosloc)
```

Comment:
Having a hard time parsing this variable name: maybe add underscores or rename

Reply:
No reply here either: same reaction to "dynamoriosloc"

Reply:
Renamed to dynamoriostaticloc

Reply:
Please use underscores: that is very hard to parse